pages/1_13/Key_Mouse_Input.md

```diff
@@ -0,0 +1,120 @@
+# Key & Mouse Input
+
+Forge uses a library called GLFW for its input events. For more in-depth information on the GLFW library, please visit their documentation website: https://www.glfw.org/documentation.html  
+
+With update .146 that forge released two days ago, it is super easy to register key and mouse input within your mod. To keep your files nice and organized I highly suggest that you make a seperate class for registering input. Name it something like InputEventHandler.
+
+### Class Setup
+
+Now that we've created our input class we have to annotate it with `@Mod.EventBusSubscriber`. This tells forge that this class wants to subscribe to the MinecraftForge EVENT_BUS when it's constructed. In order to use said annotation we have to import the following interface: `net.minecraftforge.fml.common.Mod`. Your class should now look a little something like this.
+
+```java
+package exampleinput
+
+import net.minecraftforge.fml.common.Mod
+
+@Mod.EventBusSubscriber
```

Comment:
Doesn't it need to be this?
`@Mod.EventBusSubscriber(bus=Mod.EventBusSubscriber.Bus.MOD)`

Taken from the example mod in the MDK

Reply:
I've tested this and I don't think it's necessary. The forge docs on events also don't mention this to be necessary. Correct me if I'm wrong though

Reply:
As far as I can tell, the Forge docs haven't been updated yet.

Oh right okay, reading the Java docs in Forge, it seems there's 2 options:
- Bus.FORGE
For all Forge events
- Bus.MOD
For all mod specific events (which I presume means custom events?)
Literally the description is: `The mod's event bus, to allow subscription to Mod specific events`